spec/services/bulk_upload/sales/year2022/row_parser_spec.rb

```diff
@@ -349,6 +349,19 @@ def purch_id_field
       end
     end
 
+    describe "#field_44 - 7" do # buyers organisations
+      context "when all nil" do
+        let(:attributes) { setup_section_params.merge(field_44: nil, field_45: nil, field_46: nil, field_47: nil) }
```

Comment:
Can these keys be missing entirely? Does the validation do what you expect in that instance?

Reply:
Yep, stll works in that case - though that's down to the row parser not anything I've built so don't think it's a test that should be added here